regnety/dataset/imagenet.py

```diff
@@ -127,14 +134,37 @@ def _read_tfrecs(self) -> Type[tf.data.Dataset]:
             num_parallel_calls = AUTO 
         )
 
-        ds = ds.cache("tf_cache")
-       
-        ds = ds.repeat()
-        ds = ds.batch(self.batch_size)
+        ds = ds.cache("gs://adityakane-train/cache/"+str(time.time()))
+        
+        ds = ds.batch(self.batch_size, drop_remainder=True)
```

Comment:
Why the remainder is being dropped?

Reply:
I added that we're not implementing `steps_per_epoch` for now. I'll change this once we fix that.